src/common/styles/colors.tsx

```diff
@@ -6,17 +6,19 @@ export type ColorName =
   | "blue-green"
   | "green"
   | "red";
-export type ToneLevel = 0 | 10 | 20 | 30 | 40 | 50 | 60 | 80 | 100;
+export type ToneLevel = 0 | 10 | 20 | 30 | 40 | 50 | 60 | 70 | 80 | 90;
```

Comment:
as mentioned in our call yesterday, we should not be using the color palette first.

This new color palette should be another type, and should not touch the current `ToneLevel`